shacl/Bestanddeel.ttl

```diff
@@ -23,7 +23,7 @@ bestanddeel:descriptionGroup a sh:PropertyGroup ;
     sh:order 2.0
 .
 bestanddeel:locationGroup a sh:PropertyGroup ;
-    rdfs:label "Locties"@nl ;
+    rdfs:label "Locaties"@nl ;
```

Comment:
Typo fix! :stuck_out_tongue:

Reply:
Die kunnen we meteen volgen: zodra dit in Nexus is doorgevoerd, is het label in  de UI gefixt